index.bs

```diff
@@ -4557,6 +4557,11 @@ macros:
 The number of channels of the output always equals the number of
 channels of the AudioBuffer assigned to the {{AudioBufferSourceNode/buffer}} attribute, or is one
 channel of silence if {{AudioBufferSourceNode/buffer}} is <code>null</code>.
+In addition, the {{AudioBufferSourceNode}} must output a single
+channel of silence at a rendering quantum boundary after the time at which any one of the following conditions holds:
```

Comment:
The single channel of silence is not really *at* the rendering quantum boundary, which has zero length, but *begins* at the boundary.
Perhaps
"In addition, if the buffer has more than one channel, then the {{AudioBufferSourceNode}} output must change to a single channel of silence at a rendering".